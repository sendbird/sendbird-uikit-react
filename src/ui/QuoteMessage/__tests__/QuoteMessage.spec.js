import React from 'react';
import { render } from '@testing-library/react';

import QuoteMessageItemBody from '../index';

describe('ui/QuoteMessage', () => {
  it('should do a snapshot test of the ReplyingMessageItemBody DOM', function() {
    const { asFragment } = render(
      <QuoteMessageItemBody
        message={{ sender: { nickname: 'Simon' } }}
        parentMessageType={null}
        parentMessageText="Hello nice to meet you"
        parentMessageUrl={''}
        parentMessageSender={{ nickname: 'Gabie' }}
        isByMe
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
