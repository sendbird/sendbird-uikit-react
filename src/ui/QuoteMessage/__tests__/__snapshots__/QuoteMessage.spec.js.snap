// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReplyingMessageItemBody should do a snapshot test of the ReplyingMessageItemBody DOM 1`] = `
<div
  className=" sendbird-quote-message outgoing"
  onClick={[Function]}
>
  <div
    className="sendbird-quote-message__replied-to"
  >
    <div
      className="sendbird-quote-message__replied-to__icon sendbird-icon sendbird-icon-reply sendbird-icon-color--on-background-3"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      style={
        Object {
          "height": "12px",
          "minHeight": "12px",
          "minWidth": "12px",
          "width": "12px",
        }
      }
      tabIndex="0"
    >
      <test-file-stub />
    </div>
    <span
      className="sendbird-quote-message__replied-to__text sendbird-label sendbird-label--caption-2 sendbird-label--color-onbackground-3"
    >
      Simon replied to undefined
    </span>
  </div>
  <div
    className="sendbird-quote-message__replied-message"
  />
</div>
`;
